src/shogun/features/FeaturesUtil.h

```diff
@@ -43,37 +43,25 @@ namespace internal
 
 /**
  * @brief Class FeaturesUtil for providing generic helper methods for
- * handling Shogun's feature objects for the big-testing framework.
+ * handling Shogun's feature objects.
  */
 struct FeaturesUtil
 {
 	/**
-	 * This creates a shallow copy of the feature object. It uses the same
-	 * underlying feature storage as the original object, but it clones all
-	 * the subsets.
-	 *
-	 * @param other The feature object whose shallow copy has to be created.
-	 * @return A shallow copy of the feature object.
-	 */
-	static CFeatures* create_shallow_copy(CFeatures* other);
-
-	/**
 	 * This creates a merged copy of the two feature objects.
 	 *
+	 * Please note that unlike CFeatures::create_merged_copy, this method
+	 * respects the subsets that are added to **both** these feature instances.
```

Comment:
Why would this method not be in `CFeatures`
If it should be eventually, could you create an issue or a TODO item for this?